assets/js/components/setup/compatibility-checks.js

```diff
@@ -122,35 +122,39 @@ export default class CompatibilityChecks extends Component {
 
 		if ( ! installed && installURL ) {
 			return {
-				labelHTML: __( 'Install<span class="screen-reader-text"> the helper plugin</span>', 'google-site-kit' ),
+				labelHTML: __( 'Install', 'google-site-kit' ),
+				a11yLabel: __( 'the helper plugin', 'google-site-kit' ),
 				href: installURL,
 				external: false,
 			};
 		}
 		if ( installed && ! active && activateURL ) {
 			return {
-				labelHTML: __( 'Activate<span class="screen-reader-text"> the helper plugin</span>', 'google-site-kit' ),
+				labelHTML: __( 'Activate', 'google-site-kit' ),
+				a11yLabel: __( 'the helper plugin', 'google-site-kit' ),
 				href: activateURL,
 				external: false,
 			};
 		}
 		if ( installed && active && configureURL ) {
 			return {
-				labelHTML: __( 'Configure<span class="screen-reader-text"> the helper plugin</span>', 'google-site-kit' ),
+				labelHTML: __( 'Configure', 'google-site-kit' ),
+				a11yLabel: __( 'the helper plugin', 'google-site-kit' ),
 				href: configureURL,
 				external: false,
 			};
 		}
 		return {
-			labelHTML: __( 'Learn how<span class="screen-reader-text"> to install and use the helper plugin</span>', 'google-site-kit' ),
+			labelHTML: __( 'Learn how', 'google-site-kit' ),
+			a11yLabel: __( 'to install and use the helper plugin', 'google-site-kit' ),
```

Comment:
Translatable sentences should never be split or concatenated in any way - they need to be maintained in one string like they were before. For this purpose there is `createInterpolateElement`, would be great if you had a look at that.

Reply:
Thanks for the info @felixarntz . Code has been updated to use `createInterpolateElement` :)